templates/server/puppet.conf.main.erb

```diff
@@ -0,0 +1,13 @@
+## Server config
+
+    # Puppet 3.0.x requires this in both [main] and [master] - harmless on agents
+    autosign       = <%= scope.lookupvar('::puppet::autosign') %>
```

Comment:
Since it refers to puppet 3.0.x (which we no longer support), I wonder if this is still needed. Anyone knows if this was fixed in later versions?